.github/workflows/ci-build.yml

```diff
@@ -9,16 +9,14 @@ on:
     branches:
       - master
 
-
 jobs:
-
-  test:
+   test:
```

Comment:
the number of spaces (3) is weird. Would you mind fixing it?

Reply:
ok.  I didn't even realize this changed.